phpng_xhprof.c

```diff
@@ -398,8 +403,11 @@ PHP_FUNCTION(xhprof_disable)
 {
 	if (hp_globals.enabled)
 	{
-		hp_stop(TSRMLS_C);
-		RETVAL_ARR(Z_ARR(hp_globals.stats_count));
```

Comment:
This was changed because no ref counting is being done here, leading to strangeness.